docs/evm/ethereum-vs-flow-evm.md

```diff
@@ -0,0 +1,130 @@
+
+The EVM environment has its own concept of blocks, and a single Flow block can include several EVM blocks (a one-to-many relationship). Note that since EVM blocks are formed on-chain and Flow provides fast finality, as long as the user of these events waits for Flow block finality, it doesn’t have to worry about EVM block forks, uncle chains, and other consensus-related problems on Ethereum. 
+
+### No shared memory
+
+The interaction between two environments is through atomic calls and none of the environments has access to the raw memory of the other. This maintains the security properties of each environment. Cadence can submit transactions to the EVM environment and EVM transactions can make calls to a special precompiled contract called Cadence Arch.  
+
+### No new native token
+
+EVM on Flow uses the same native token as Cadence (FLOW token). No new token is minted at the genesis block of EVM and all the tokens have to be bridged over from the Cadence side into the EVM side. To facilitate this a native bridge is provided by the EVM contract. 
+
+### EVM equivalency
+
+Under the hood, EVM on Flow uses [the standard EVM implementation](https://github.com/ethereum/go-ethereum) and regularly applies updates through Flow’s height-coordinated updates. This means anything that can run on Ethereum after the Shanghai fork can run on Flow EVM. This means many useful EIPs such as [EIP-1014](https://eips.ethereum.org/EIPS/eip-1014), [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), … are supported automatically.
+
+Yet a small set of differences between EVM on Flow and Ethereum might be seen (mostly of the nature of extension) for two reasons: 
+
+- A set of extensions has been added to ensure seamless and easy interaction between the two environments.
+- EVM on Flow is secured by the Flow network and benefits from its nice properties. For example, Flow supports fast block production and finalization, making issues like handling uncle chains irrelevant.
+
+### Gateways
+
+As mentioned earlier EVM on Flow doesn’t have its own consensus and runs on top of the Flow network. There is no new node type for the EVM on Flow. Anyone who wants to run the network can stake tokens and participate as Flow nodes.
+
+Gateway is a new software package provided by the Flow Foundation. It communicates with the Flow access nodes and can be run by anyone (unstaked). It serves two purposes:
```

Comment:
Lets reformat this a bit as the two purposes we highlight have headers that are the same size as the following 'Censorship resistance and MEV protection' section making it visually harder to follow

Reply:
updated